packages/ui/src/proposals/components/AddProposalButton/AddProposalButton.stories.tsx

```diff
@@ -0,0 +1,13 @@
+import { Meta, Story } from '@storybook/react'
+import React from 'react'
+
+import { AddProposalButton } from './AddProposalButton'
+
+export default {
+  title: 'Proposals/AddProposalButton',
+  component: AddProposalButton,
+} as Meta
+
+const Template: Story = () => <AddProposalButton />
+
+export const Default = Template.bind({})
```

Comment:
I think that for a simple button the story is not needed. You can remove that file.